ts-rs/src/export.rs

```diff
@@ -102,7 +116,13 @@ pub(crate) fn export_type_to<T: TS + ?Sized + 'static, P: AsRef<Path>>(
     static FILE_LOCK: Mutex<()> = Mutex::new(());
 
     #[allow(unused_mut)]
-    let mut buffer = export_type_to_string::<T>()?;
+    // TODO: this generates imports, but disregards 'path'!
+    //       In general, imports won't work anymore if users call `TS::export_to`, and manually 
+    //       specify each file instead of using `#[ts(export_to = "..")]`. 
+    //       However, if users only do that with types that no other type depends on, then imports
+    //       should still work! To do that, we'd need to pass `path` down until we get it into 
+    //       `generate_imports`.
```

Comment:
TL;DR:
> `TS::export_to`: Imports are generated using the "default" `TS_RS_EXPORT_DIR` + `export_to` path, instead of the one the user provided

I have no clue how we'd ever fix this